cmd/earthly/main.go

```diff
@@ -1566,6 +1566,10 @@ func (app *earthlyApp) run(ctx context.Context, args []string) int {
 				"Check your git auth settings.\n" +
 					"Did you ssh-add today? Need to configure ~/.earthly/config.yml?\n" +
 					"For more information see https://docs.earthly.dev/guides/auth\n")
+		} else if strings.Contains(err.Error(), "failed to compute cache key") && strings.Contains(err.Error(), ": not found") {
+			var re = regexp.MustCompile(`".*?"`)
```

Comment:
This regex feels a bit fragile. Would something like https://go.dev/play/p/Y57YqC0U-0B be better?

Reply:
Addressed. Now logging full original error if regex doesn't match.